src/modules/date/index.ts

```diff
@@ -174,22 +177,34 @@ export class SimpleDateModule extends SimpleModuleBase {
      */
     to: string | Date | number;
   }): Date {
+    if (!options || !options.from || !options.to) {
```

Comment:
I just noticed that this might cause issues with UTC epoch/`0`.

So maybe we should use a nullish check instead?

```suggestion
    if (!options || options.from == null || options.to == null) {
```
Or
```suggestion
    if (options == null || options.from == null || options.to == null) {
```

Sorry for the extra work.

Reply:
did this, and added an extra test for date 0